examples/DimensionReduction/emulate_sample_linlinexp.jl

```diff
@@ -0,0 +1,142 @@
+            (decorrelate_structure_mat(; retain_var = 1.0), "out"),
+        ]
+        encoder_schedules_li = [
+            [
+                (decorrelate_structure_mat(; retain_var = 1.0), "in_and_out"),
+                (LikelihoodInformed(nothing, nothing, nothing, (:dimension, dim), α, :linreg, false), "in"),
+            ] for α in αs
+        ]
+
+        em_ref = Emulator(
+            NoEmulation(param -> forward_map(param, model)),
+            Utilities.get_training_points(ekpobj, min_iter:max_iter);
+            encoder_schedule = encoder_schedule_ref,
+            encoder_kwargs = (; prior_cov = cov(prior_obj), obs_noise_cov = obs_noise_cov),
+        )
+
+        em_decorrelate = Emulator(
+            NoEmulation(param -> forward_map(param, model)),
+            Utilities.get_training_points(ekpobj, min_iter:max_iter);
+            encoder_schedule = encoder_schedule_decorrelate,
+            encoder_kwargs = (; prior_cov = cov(prior_obj), obs_noise_cov = obs_noise_cov),
+        )
+
+        ems_li = [
+            Emulator(
+                NoEmulation(param -> forward_map(param, model)),
+                Utilities.get_training_points(ekpobj, min_iter:max_iter);
+                encoder_schedule = encoder_schedule,
+                encoder_kwargs = (;
+                    prior_cov = cov(prior_obj),
+                    obs_noise_cov = obs_noise_cov,
+                    samples_in = ekp_samp[α][1],
+                    samples_out = ekp_samp[α][2],
+                    observation = y,
+                ),
+            ) for (encoder_schedule, α) in zip(encoder_schedules_li, αs)
+        ]
+
+        post_means = reshape(true_parameter, input_dim, 1)
+        post_covs = []
+        for em in vcat(em_ref, em_decorrelate, ems_li...)
+            u0 = rand(MvNormal(mean(prior_obj), cov(prior_obj)))
+            mcmc = MCMCWrapper(RWMHSampling(), y, prior_obj, em; init_params = u0)
+            new_step = optimize_stepsize(mcmc; init_stepsize = 0.05, N = 2000, discard_initial = 0)
+
+            println("Begin MCMC - with step size ", new_step)
+            num_chains = 16
+            mcmc = MCMCWrapper(RWMHSampling(), y, prior_obj, em; init_params = [u0 for _ in 1:num_chains])
+            chain = MarkovChainMonteCarlo.sample(
+                mcmc,
+                MCMCThreads(),
+                40_000,
+                num_chains;
+                chain_type = Chains,
+                stepsize = new_step,
+                discard_initial = 5_000,
+            )
+
+            posterior = MarkovChainMonteCarlo.get_posterior(mcmc, chain)
+
+            post_mean = mean(posterior)
+            post_cov = cov(posterior)
+
+            post_means = hcat(post_means, reshape(post_mean, input_dim, 1))
+            push!(post_covs, post_cov)
+        end
+
+        all_errs[dim_i, :] =
+            [norm(post_means[:, 2] - v) / norm(post_means[:, 2]) for v in eachcol(post_means[:, 3:end])]'
```

Comment:
Why is the reference emulator not used here? Otherwise this is just showing the size of difference to a PCA-truncation mean.

PS. Running the example gave:

```5×6 Matrix{Float64}:
 0.771104  0.724079  0.686967  0.675671  0.720269  0.687709
 0.397877  0.567556  0.629621  0.656653  0.666858  0.66542
 0.40432   0.393665  0.350307  0.287113  0.297794  0.238406
 0.436975  0.370324  0.34126   0.278414  0.262382  0.233595
 0.376547  0.295315  0.151808  0.147098  0.094317  0.118197
```